packages/bsky/src/index.ts

```diff
@@ -161,10 +168,12 @@ export class BskyAppView {
     await events.once(server, 'listening')
     const { port } = server.address() as AddressInfo
     this.ctx.cfg.assignPort(port)
+    await this.ctx.featureGates.start()
     return server
   }
 
   async destroy(): Promise<void> {
+    this.ctx.featureGates.destroy()
     await this.terminator?.terminate()
```

Comment:
Swapping these allows features gates to continue to be applied while requests drain from the server.
```suggestion
    await this.terminator?.terminate()
    this.ctx.featureGates.destroy()
```

Reply:
Oh awesome, makes sense